### PR TITLE
Project cleanup

### PR DESCRIPTION
**Pull request by sharwell**
Recommended review strategy is commit-by-commit.

💡 Please do not rebase or squash this change during the merge.

📝 I am planning to submit a follow-up PR to improve workflow for new users, but I am submitting this one separately so the review is simpler.

### CODE REVIEW

File: src/VAP/BGSObjectDetector/BGSObjectDetector.csproj [hunk 1]
```diff
@@ -1,16 +1,11 @@
...
-    <OutputType>Library</OutputType>
     <TargetFramework>netcoreapp2.2</TargetFramework>
-    <ApplicationIcon />
-    <StartupObject />
   </PropertyGroup>
 
   <ItemGroup>
     <PackageReference Include="OpenCvSharp4" Version="4.0.0.20190108" />
-    <PackageReference Include="OpenCvSharp4.runtime.ubuntu.18.04-x64" Version="4.0.0.20190108" />
-    <PackageReference Include="OpenCvSharp4.runtime.win" Version="4.0.0.20190108" />
```
Comment (sharwell) [inline src/VAP/BGSObjectDetector/BGSObjectDetector.csproj:13]: 📝 These are only needed for executables. Removing the references from libraries simplifies version management a bit.

File: .gitattributes [hunk 2]
```diff
@@ -0,0 +1 @@
+* text=auto
```
Comment (sharwell) [inline .gitattributes:1]: 📝 This change ensures line end-of-line normalization does not cause behavior differences between users on Windows and users on other platforms.

File: src/VAP/YoloWrapper/YoloWrapper.csproj [hunk 3]
```diff
@@ -1,61 +1,14 @@
...
-    <Platform Condition=" '$(Platform)' == '' ">AnyCPU</Platform>
-    <ProjectGuid>{992D9B38-FCC1-406A-92A2-6C1B5A310CCF}</ProjectGuid>
-    <OutputType>Library</OutputType>
-    <AppDesignerFolder>Properties</AppDesignerFolder>
-    <RootNamespace>YoloWrapper</RootNamespace>
-    <AssemblyName>YoloWrapper</AssemblyName>
-    <TargetFrameworkVersion>v4.6.1</TargetFrameworkVersion>
-    <FileAlignment>512</FileAlignment>
-    <Deterministic>true</Deterministic>
+    <TargetFramework>netcoreapp2.2</TargetFramework>
```
Comment (sharwell) [inline src/VAP/YoloWrapper/YoloWrapper.csproj:5]: 📝 This is a change from the original project which used **net461**, but the rest of the solution used **netcoreapp2.2** so this seemed like the intended behavior.

**Comment by ycshu086** [comment]
Thank you for the cleanup. :)

**Pull request merged**